src/video/x11/SDL_x11modes.c

```diff
@@ -220,10 +220,11 @@ static SDL_bool SetXRandRModeInfo(Display *display, XRRScreenResources *res, RRC
                 mode->w = info->width;
                 mode->h = info->height;
             }
-            mode->refresh_rate = CalculateXRandRRefreshRate(info);
+            mode->refresh_rate_numerator = CalculateXRandRRefreshRate(info);
```

Comment:
I'm not positive about this but I wonder if a real numerator and denominator can be exposed here, since `CalculateXRandRRefreshRate` seems to do some calculations on values that look like that?

Reply:
updated, seems ok to me